### DISCUSSION TITLE
When will persistQueryClient/createAsyncStoragePersistor not be experimental?

### DISCUSSION THREAD

**Discussion by henrymoulton** [TanStack/query#3179] [Q&A]
I noticed that persistQueryClient has been live for almost a year now, would be good to know what it might take to not be considered "Experimental".

I'd be happy to try and contribute if there's any tasks to make it not marked as experimental anymore, but I couldn't find a roadmap.

Potentially relevant:
https://github.com/tannerlinsley/react-query/discussions/2649
https://github.com/tannerlinsley/react-query/pull/3141
https://github.com/tannerlinsley/react-query/pull/2864

**Comment by TkDodo** [comment]
in v4:

https://react-query-alpha.tanstack.com/guides/migrating-to-react-query-4#persistqueryclient-and-the-corresponding-persister-plugins-are-no-longer-experimental-and-have-been-renamed
